content/applications/services/helpdesk/overview/stages.rst

```diff
@@ -0,0 +1,113 @@
+
+Add email and SMS templates to stages
+=====================================
+
+When an :guilabel:`Email Template` is added to a stage, an email is automatically sent to the
+customer when a ticket reaches that specific stage in the pipeline. Likewise, adding an
+:guilabel:`SMS Template` triggers an SMS text message to send to the customer.
+
+.. important::
+   SMS Text Messaging is an :doc:`In-App Purchase (IAP) </applications/essentials/in_app_purchase/>`
+   service that requires prepaid credits to work. Refer to `SMS Pricing FAQ
+   <https://iap-services.odoo.com/iap/sms/pricing>`_ for additional information.
+
+To select an existing email template, select it from the :guilabel:`Email Template` field. Click on
+the :icon:`oi-arrow-right` :guilabel:`(right arrow)` icon to the right of the field to edit the
+chosen template.
+
+To create a new template, click the field, and enter a title for the new template. Then, select
+:guilabel:`Create and edit` from the drop-down menu that appears, and complete the form details.
+
+Follow the same steps to select, edit, or create an :guilabel:`SMS Template`.
+
+.. image:: stages/sms-template.png
+   :alt: View of an SMS template setup page in Odoo Helpdesk.
+
+.. seealso::
+   :doc:`/applications/general/companies/email_template`
+
+Assign stages to a team
+=======================
+
+Make a selection in the :guilabel:`Helpdesk Teams` field on the :guilabel:`Stages` form. More than
+one team may be selected, since the same stage can be assigned to multiple teams.
+
+Fold a stage
+============
+
+By default, stages are unfolded in the Kanban view of either tickets dashboard: :guilabel:`My
+Tickets` (:menuselection:`Helpdesk app --> Tickets --> My Tickets`) or :guilabel:`All Tickets`
+(:menuselection:`Helpdesk app --> Tickets --> All Tickets`).
+
+Tickets in an unfolded stage are visible in the pipeline under the stage name, and are considered
+*open*.
+
+Stages can be configured to be folded in the Kanban view of a tickets page (:guilabel:`My Tickets`
+or :guilabel:`All Tickets`).
+
+The name of the folded stages are still visible, though the tickets in the stage are no longer
+immediately visible.
+
+To fold a stage, check the :guilabel:`Folded in Kanban` box on the :guilabel:`Stages` form.
+
+.. warning::
+   Tickets that reach a *folded* stage are considered *closed*. Closing a ticket before the work is
+   completed can result in reporting and communication issues. This setting should **only** be
+   enabled for stages that are considered *closing* stages.
+
+Temporarily fold a stage
+------------------------
+
+Stages can be temporarily folded in the Kanban view of the tickets pipeline, as well.
+
+View a specific team's pipeline by navigating to :menuselection:`Helpdesk app`, and clicking the
+team's Kanban card.
+
+Select a stage to fold temporarily, then click the :icon:`fa-gear` :guilabel:`(gear)` icon, and
+select :guilabel:`Fold` from the drop-down menu.
```

Comment:
```suggestion
Select a stage to fold temporarily, then click the :icon:`fa-gear` :guilabel:`(Actions)` icon, and
select :guilabel:`Fold` from the drop-down menu.
```

Sam just edited this for me in a doc, and we are using this from now on for the gear icon.

Reply:
This is actually referring to two different things. The "Actions" icon is only referring to the Actions menu. In this case, the gear icon is used on the stage in the Kanban view to open the Settings menu. So I do not think it makes sense to call it the Actions icon.

Reply:
@jero-odoo and @larm-odoo, thanks for discussing this distinction. I agree with Jess, that the gear icon should only be labeled as the "Actions" icon if it is referring to the Actions menu.